### PR TITLE
Fix check target for Unix-like environment

### PR DESCRIPTION
**Pull request by akihikodaki**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -407,15 +405,8 @@ else
...
-	mkdir -p $(DESTDIR)/$(LIBDIR)
-ifeq ($(CAPSTONE_SHARED),yes)
-	$(INSTALL_LIB) $(LIBRARY) $(DESTDIR)/$(LIBDIR)
-ifneq ($(VERSION_EXT),)
-	cd $(DESTDIR)/$(LIBDIR) && \
-	rm -f lib$(LIBNAME).$(EXT) && \
-	ln -s lib$(LIBNAME).$(VERSION_EXT) lib$(LIBNAME).$(EXT)
-endif
-endif
+	mkdir -p $(LIBDIR)
```
Comment (aquynh) [inline Makefile]: why did you change `mkdir -p $(DESTDIR)/$(LIBDIR)` to `mkdir -p $(LIBDIR)` here?

**Comment by akihikodaki** [comment]
ec4075377e940d6e981f934ed97c6664ff724956: Fixed https://github.com/aquynh/capstone/pull/770#discussion-diff-78280663

**Comment by aquynh** [comment]
merged, thanks!
